server/src/main/resources/db/migration/V1_376__add_candidate_family_members_form.sql

```diff
@@ -0,0 +1,12 @@
+insert into candidate_form (name, description)
+values ('FamilyDocForm',
+        'Enter details of your eligible relocating family members.');
+
+
+insert into task (name, display_name, task_type, description, days_to_complete,
+                  created_by, created_date, candidate_form_id)
+values ('familyDocFormTask','Enter details of your eligible relocating family members',
```

Comment:
Display name should be shorter than this. This is more like a description. This name will appear in the drop down of tasks to be completed. I suggest "Complete relocating dependant data"